Resources/Prototypes/Entities/Mobs/Species/human.yml

```
@@ -26,107 +26,1214 @@
+## Based
+# New
     femaleDisplacements:
       jumpsuit:
         sizeMaps:
           32:
-            sprite: Mobs/Species/Human/displacement.rsi
+            sprite: _Sunrise/Mobs/Species/Human/New/Basic/displacement.rsi
             state: jumpsuit-female
-      jumpsuit-body-slim:
+
+# Old
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)